library/ssl_cli.c

```diff
@@ -59,8 +59,7 @@
 #if defined(MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED)
 static int ssl_conf_has_static_psk( mbedtls_ssl_config const *conf )
 {
-    if( conf->psk_identity     == NULL ||
-        conf->psk_identity_len == 0     )
+    if( conf->psk_identity     == NULL )
```

Comment:
Minor: Indentation is off now that the second line has been removed.

Reply:
Yup. Fixed.